.github/workflows/CI.yml

```diff
@@ -30,6 +30,15 @@ jobs:
         os:
```

Comment:
```suggestion
        depwarn: ['']
        os:
```
Is this maybe needed for the new jobs to get created?

Reply:
You could also just add a separate job, if you don't want to do this dance.